ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-aro.yaml

```diff
@@ -18,37 +18,18 @@ base_images:
   tests-private-preupg:
     name: tests-private
     namespace: ci
-    tag: "4.12"
+    tag: "4.13"
   tools:
-    name: "4.12"
+    name: "4.14"
```

Comment:
I think `tools`, `cli` is supposed to be set to the source version (e.g: the job is running 4.13 ->4.14 upgrade, the version should be set to 4.13, but not 4.14), and sounds `ansible` base image can be dropped now.